common/projectconfig/projectconfig.go

```diff
@@ -84,6 +85,9 @@ func loadFile(path string) (Config, error) {
 	config := Config{}
 	md, err := toml.DecodeFile(path, &config)
 	if err != nil {
+		if errors.Is(err, os.ErrNotExist) {
```

Comment:
This allows `ftl config` to operate when the `ftl-project.toml` file doesn't exist yet.